### PR TITLE
Add API Response Model in LoanAccountFragment #385 Resolved

### PR DESCRIPTION
**Pull request by mayank-kgp**
Please make sure these boxes are checked before submitting your pull request - thanks!
- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.
- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything
- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@therajanmaurya  Please review it once.

File: mifosng-android/src/main/java/com/mifos/objects/accounts/loan/OverdueCharges.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.mifos.objects.accounts.loan;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by mayankjindal on 22/10/16.
+ */
+
+public class OverdueCharges implements Parcelable {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/objects/accounts/loan/OverdueCharges.java]: remove this POJO, If you do not using it.

**Comment by therajanmaurya** [comment]
@mayank-kgp one more thing, Please remove **private** identifier of fields because DBFlow 
genrating Database boilerPlate code that needs to access POJO fields, Some time it unable to access the field due to bad name getter setter etc. 
So, stay with default access identifier.

**Comment by mayank-kgp** [comment]
@therajanmaurya  Should I remove those private fields also which were already there and not written by me?

**Comment by therajanmaurya** [comment]
Yes

**Comment by mayank-kgp** [comment]
@therajanmaurya Should I remove private fields in only those pojo which have dbflow implementation or should i remove in all files?

**Comment by therajanmaurya** [comment]
I think all because we have to make a similar thing for the whole project but wait for @ishan1604. 

@ishan1604 What you think about the **public** fields of POJOS.

DBFlow needs the **public** fields but we are not saving everything in Database that why some fields are not annotated with DBFlow annotation. we should make then **private** or not.

**Comment by mayank-kgp** [comment]
@therajanmaurya waiting for @ishan1604  reply.

**Comment by mayank-kgp** [comment]
@therajanmaurya  @ishan1604 Please look into this.

**Comment by mayank-kgp** [comment]
@therajanmaurya Waiting for response.

**Comment by therajanmaurya** [comment]
You can remove the ```public``` ```private```

**Comment by mayank-kgp** [comment]
@therajanmaurya  I have removed the private identifiers in my PR.
Please review it and let me know if any further change is required.

**Comment by therajanmaurya** [comment]
Hi @ishan1604 This PR looks good Please merge

**Comment by mayank-kgp** [comment]
@ishan1604  Please have a look at it.

**Pull request merged**